src/libraries/System.Drawing.Common/src/System/Drawing/FontFamily.cs

```diff
@@ -67,7 +67,7 @@ internal FontFamily(string name, bool createDefaultOnFail)
 
```

Comment:
`FontCollection fontCollection` above is nullable

Reply:
Good catch, thanks